crosstool/cc_toolchain_config.bzl

```
@@ -1847,7 +1847,7 @@ def _impl(ctx):
                     ACTION_NAMES.objc_compile,
                     ACTION_NAMES.objcpp_compile,
                 ],
-                flag_groups = [flag_group(flags = ["DEBUG_PREFIX_MAP_PWD=."])],
+                flag_groups = [flag_group(flags = ["-fdebug-prefix-map=__BAZEL_EXECROOT__=."])],
```

```suggestion
                flag_groups = [flag_group(flags = ["-fdebug-prefix-map=__BAZEL_EXECUTION_ROOT__=."])],
```

Technically `execroot` and `execution root` are different, and I like these to be named to match `bazel info execution_root`.